parts/k8s/addons/cloud-node-manager.yaml

```diff
@@ -148,16 +182,40 @@ spec:
         command:
         - /cloud-node-manager.exe
         - --node-name=$(NODE_NAME)
+        - --kubeconfig=C:\k\config
```

Comment:
Did we want this new `--kubeconfig` configuration to be outside of the Azure Stack block?

Reply:
Based on my test on Azure, the windows node will also need this config to launch the cloud node manager pod. The default IP in external-cloud-provider is not working.

Reply:
Sample error: 
`
E0902 16:06:16.057842    1344 nodemanager.go:183] Error monitoring node status: Get "https://10.0.0.1:443/api/v1/nodes?fieldSelector=metadata.name%3D2356k8s010&resourceVersion=0": dial tcp 10.0.0.1:443: connectex: A connection attempt failed because the connected party did not properly respond after a period of time, or established connection failed because connected host has failed to respond.
`